### ISSUE TITLE
[BUG] Mobile site menu item hides behind Title

### ISSUE THREAD

**Issue by KashifZA** [katznboyz1/youtube-dl-server#5]
So when you visit site on chrome mobile (ios) and expand hamburger menu on the history page, it hides behind the title page. Also the floating menu doesn't have a solid colour so it looks like it bleeds into background which can make it hard to read.

This appear to happen for the History page which has a table header. For the other pages, the page shifts down (iOS -Phone & iPad with Chrome,)

**Comment by renamedquery** [comment]
Yes I have noticed this too! I am not sure what to do at the moment because my mobile web development skills are not great, but I'll look into how to get the site to load properly. Expect an update in a few days.
